cypress/support/cypress-commands/lifecycle.js

```diff
@@ -477,3 +478,67 @@ Cypress.Commands.add('lifecycleSchemaChecks', (response, state) => {
   };
   cy.validationChecksForResponseAndSchemaResult(response, false, apiSchemaResult, false);
 });
+
+/**
+ * @module commands
+ * @function validateVisibilityState
+ * @description To validate app visibility for different lifecycle states
+ * @param {String} state - To check visibilityState of state
+ * @example
+ * validateVisibilityState('foreground');
+ */
+function validateVisibilityState(state) {
+  // Fetching the visibilityState for the states from env.
+  const visibilityState = Cypress.env('visibilityState');
```

Comment:
"visibilityState" is used in multiple places and should be in a constant.

Reply:
Moved to "visibilityState" to contants